src/main/drivers/system_stm32f7xx.c

```diff
@@ -162,6 +188,10 @@ void systemInit(void)
 {
     checkForBootLoaderRequest();
 
+    //  Mark ITCM-RAM as read-only
+    LL_MPU_ConfigRegion(LL_MPU_REGION_NUMBER0, 0, RAMITCM_BASE, LL_MPU_REGION_SIZE_16KB | LL_MPU_REGION_PRIV_RO_URO);
+    LL_MPU_Enable(LL_MPU_CTRL_PRIVILEGED_DEFAULT);
```

Comment:
Enable the MPU with default memory map for unspecified regions, effectively changing nothing for any memory except ITCM-RAM.